common/djangoapps/student/tests/test_login.py

```diff
@@ -273,6 +273,43 @@ def test_single_session(self):
         self.assertEqual(response.status_code, 302)
 
     @patch.dict("django.conf.settings.FEATURES", {'PREVENT_CONCURRENT_LOGINS': True})
+    def test_single_session_with_no_user_profile(self):
+        """
+        Assert that user login with cas (Central Authentication Service) is
+        redirect to dashboard in case of lms or upload_transcripts in case of
+        cms
+        """
+        user = UserFactory.build(username='tester', email='tester@edx.org')
+        user.set_password('test_password')
+        user.save()
+
+        creds = {'email': 'test@edx.org', 'password': 'test_password'}
```

Comment:
Are these creds supposed to represent the user created above? If so, the email address does not match.